sql/sys_vars.cc

```diff
@@ -6520,6 +6521,37 @@ static Sys_var_charptr Sys_wsrep_allowlist(
        READ_ONLY GLOBAL_VAR(wsrep_allowlist), CMD_LINE(REQUIRED_ARG),
        DEFAULT(""));
 
+static Sys_var_charptr_fscs Sys_wsrep_buffered_error_log_filename(
+    "wsrep_buffered_error_log_filename", "Filename of the buffered error log",
+    PREALLOCATED GLOBAL_VAR(wsrep_buffered_error_log_filename), CMD_LINE(REQUIRED_ARG),
+    DEFAULT(0), NO_MUTEX_GUARD, NOT_IN_BINLOG,
+    ON_CHECK(wsrep_buffered_error_log_filename_check),
+    ON_UPDATE(wsrep_buffered_error_log_filename_update));
+
+static Sys_var_ulonglong Sys_wsrep_buffered_error_log_buffer_size(
```

Comment:
Why would want to set the buffer size? And what would be a good number?
Can't we just pick a fixed number, based on some basic performance testing?`

Reply:
Naturally we could, buffer size configuration depends wsrep-debug level, higher lever produces more messages. In wsrep-debug it is around 2000kb/s (sysbench).